src/net_processing.cpp

```diff
@@ -1590,6 +1590,11 @@ void static ProcessGetData(CNode* pfrom, const CChainParams& chainparams, CConnm
             if (mi != mapRelay.end()) {
                 connman->PushMessage(pfrom, msgMaker.Make(nSendFlags, NetMsgType::TX, *mi->second));
                 push = true;
+                // Once a peer requests GETDATA for a txn, we deem initial broadcast a success
```

Comment:
The comment is incorrect, since a tx is *not* removed when it is not sent, e.g. because it was not found.

Reply:
I don't fully understand what you mean here. Are you pointing out that there are other ways the transaction gets removed from the set?

I updated to: "We intepret processing a GETDATA for a transaction as a successful initial broadcast and remove it from our unbroadcast set." 

Does that seem reasonable? Or am I still missing the point?